travis_after_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]
+then
+docker exec openmrs-referenceapplication cat .OpenMRS/openmrs.log
```

Comment:
Okay, I will add it in follow-up. After reading this https://docs.travis-ci.com/user/customizing-the-build#Build-Timeouts I think that Travis is killing our build after 50 minutes(with VM startup included) no matter what we put into config. 

Maybe we could find some way to detect if server started successfully and stop test suite much earlier If it failed? That would save our time, resources, and finally we would see server logs...